lib/output-files.js

```diff
@@ -49,6 +61,12 @@ class OutputFiles {
     for (var i = 0; i < this.coverageInfo.length; i++) {
       var path = this.rewritePath(this.coverageInfo[i].url)
       this.coverageInfo[i].url = path
+
+      // create new dir structure for modules
+      if (!fs.existsSync(pathLib.parse(path).dir)) {
```

Comment:
what's our reasoning for requiring the full path for a URL but not for a `file:///`? seems like we should potentially have this logic in both cases?

Reply:
Reasoning? Trying to change as little as possible in one go :) Fixed.